src/database/DataTypes/RequisitionItem.js

```diff
@@ -85,6 +85,12 @@ export class RequisitionItem extends Realm.Object {
    * @return  {number}
    */
   get suggestedQuantity() {
+    if (this.requisition.isResponse && this.requisition.program) {
+      const daysInPeriod = this.numberOfDaysInPeriod ?? 1;
+      const dailyConsumption = this.outgoingStock / (daysInPeriod - this.daysOutOfStock);
+      return dailyConsumption * this.requisition.daysToSupply - this.stockOnHand;
```

Comment:
This is getting messy - essentially needing to update the usage, but also accounting for days out of stock in the usage.

Possibly a better way to do this would be to modify the main usage calculation: https://github.com/openmsupply/mobile/blob/a9ad1d9c2b48cc305804d4caefbd90146fccbb2a/src/utilities/dailyUsage.js#L75 to account for DOS and a passed consumption.. or something.

Alternatively: create a third usage calculation in this file

Reply:
Have adjusted this and gone with the adding a third usage calc which is updated and used when editing either the outgoing stock or days out of stock which I think has simplified the logic a bit!